rules/Removing/NodeManipulator/ComplexNodeRemover.php

```diff
@@ -310,4 +185,31 @@ private function resolvePropertyFetchFromDimFetch(Expr $expr): array
 
         return $propertyFetches;
     }
+
+    private function clearParamFromConstructor(ClassMethod $classMethod, Variable $assignedVariable): void
+    {
+        // is variable used somewhere else? skip it
+        $variables = $this->betterNodeFinder->findInstanceOf($classMethod, Variable::class);
```

Comment:
there is `findInstancesOfInFunctionLikeScoped` for it:

```suggestion
        $variables = $this->betterNodeFinder->findInstancesOfInFunctionLikeScoped($classMethod, [Variable::class]);
```

so no need to get from inner function stmts.